### PR TITLE
Fix: 1526 hide nav bar when user IDIR is unauthorized

### PR DESCRIPTION
**Pull request by BCerki**
card: https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-cif/1526

### CODE REVIEW

**Review by Sepehr-Sobhani — CHANGES_REQUESTED** [review]
Looks great 🤩  just some minor comments.

File: app/tests/unit/components/Layout/DefaultLayout.test.tsx [hunk 1]
```diff
@@ -59,7 +59,28 @@ describe("The DefaultLayout component", () => {
...
+          userGroups: ["cif_admin"],
+        };
+      },
+    });
+    componentTestingHelper.renderComponent();
+
+    expect(screen.getByText("CleanBC Industry Fund")).toBeVisible();
+    expect(screen.queryByText("Home")).toBeNull();
+    expect(screen.queryByText("Projects")).toBeNull();
+  });
```
[the remark this answers is not in the document]
Comment (BCerki) [reply app/tests/unit/components/Layout/DefaultLayout.test.tsx:81]: There's no `unauthorized_idir` component, so I don't think we need an `unauthorized_idir` file. I did move this into a `Navigation` component test though

[the remark this answers is not in the document]
Comment (Sepehr-Sobhani) [reply app/tests/unit/components/Layout/DefaultLayout.test.tsx]: We have `unauthorized_idir.tsx` and `unauthorized_idir.test.tsx` files. wouldn't it be better to move tests there? just a suggestion though.

[the remark this answers is not in the document]
Comment (BCerki) [reply app/tests/unit/components/Layout/DefaultLayout.test.tsx]: Missed that somehow, I'm going to do this PR slightly differently then

**Review by Sepehr-Sobhani — APPROVED** [review]
🥳

**Pull request merged**